stdlib/asm/sys.masm

```diff
@@ -0,0 +1,26 @@
+proc.stack_cleanup_proc.16
+    popw.local.0
+    popw.local.1
+    popw.local.2
+    popw.local.3
+    push.env.sdepth
+    push.16
+    u32gt.unsafe
+    while.true
+        dropw
+        push.env.sdepth
+        push.16
+        u32gt.unsafe
+    end
+    loadw.local.3
+    swapw.3
+    loadw.local.2
+    swapw.2
+    loadw.local.1
+    swapw.1
+    loadw.local.0
+end
+
+export.stack_cleanup
```

Comment:
Let's rename this to `finalize_stack`. Also, would be great to add brief documentation comments to the function.